app/views/layouts/_header.html.erb

```diff
@@ -9,6 +9,17 @@
 <% end %>
 <p class="facebook-summary" style="display:none;"><%= t('layout._header.summary') %></p>
 
+<script>
```

Comment:
If this is on every page, can it go in `app/assets/javascripts/setup.js` instead of inline?